src/foundry/common/constants.mjs.d.ts

```diff
@@ -372,16 +372,19 @@ export const TEXT_ANCHOR_POINTS: Readonly<{
 }>;
 export type TEXT_ANCHOR_POINTS = ValueOf<typeof TEXT_ANCHOR_POINTS>;
 
-/**
- * Define the valid occlusion modes which an overhead tile can use
- */
-export const TILE_OCCLUSION_MODES: Readonly<{
+export const OCCLUSION_MODES: Readonly<{
   NONE: 0;
   FADE: 1;
-  // ROOF: 2;  This mode is no longer supported so we don't use 2 for any other mode
+  // ROOF: 2,  This mode is no longer supported so we don't use 2 for any other mode
```

Comment:
Put it back to `Roof: 2;` - this is the TypeScript syntax. It's probably not the exact comment Foundry wrote.

Reply:
That's actually an exact copy of the Foundry comment, but I'll update it anyways